packages/calcite-components/src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -159,13 +161,15 @@ export class InputTimePicker
   @Prop({ reflect: true, mutable: true }) open = false;
 
   @Watch("open")
-  openHandler(value: boolean): void {
+  openHandler(open: boolean): void {
+    onToggleOpenCloseComponent(this);
```

Comment:
I know we'll no longer be modifying `open` based on other props, but until then, can we move this to the end of the 'closing block' below? Otherwise, events will fire when the component is blocking the component from being opened.